CMakeLists.txt

```diff
@@ -484,49 +484,47 @@ if(AVIF_CODEC_LIBRARIES MATCHES vmaf)
 endif()
 
 option(AVIF_BUILD_APPS "Build avif apps." OFF)
-if(AVIF_BUILD_APPS)
+option(AVIF_BUILD_TESTS "Build avif tests." OFF)
+
+if(AVIF_BUILD_APPS OR AVIF_BUILD_TESTS)
+    # Not all targets depending on avifappsshared, such as tests, need png and jpeg.
+    # However it is simpler this way and likely that a test environment provides these.
     find_package(ZLIB REQUIRED)
     find_package(PNG REQUIRED)
     find_package(JPEG REQUIRED)
 
-    add_executable(avifenc
-        apps/avifenc.c
-
+    add_library(avifappsshared STATIC # Static so it does not have to be installed.
```

Comment:
Nit: Shorten the library name to "avifapps" or "avif_apps". "avif_apps" is the name I used in the BUILD file in Google's internal repository. It would be good to be consistent.